test/index.js

```diff
@@ -29,7 +29,13 @@ describe('Intercom', function(){
     settings = {
       appId: 'fcxywseo',
       apiKey: '9d068fa090d38be4c715b669b3f1370f76ac5306',
-      collectContext: false
+      collectContext: false,
+      blacklisted: {
+        stringifyMe: 'stringify',
+        dropMe: 'drop',
+        flattenMe: 'flatten'
+      },
+      defaultMethod: 'flatten'
```

Comment:
a reminder here that in metadata this option should be a `select` component since we want a dropdown menu rather than text input.